### DISCUSSION TITLE
Support exclude globs

### DISCUSSION THREAD

**Discussion by greg-hellings** [ansible/ansible-lint#1424] [Ideas]
exclude_paths does not currently seem to support globs. It should.

**Comment by brlin-tw** [comment]
Second this.  I would like to exclude our roles(all named in a similar pattern) which are linted separately without cluttering the role user's lint results.  Something like this:

```yaml
exclude_paths:
  - Ansible/playbooks/roles/proprietary.*/
```

Globbing support would be helpful that we won't have to enumerate them one at a time.

**Reply by ssbarnea** [reply]
If I remember well, exclude paths work as prefixes right now which should be ok-ish for the moment. Still, as we do have glob support (via wcmatch), I would not mind reviewing a PR.

**Comment by lahwaacz** [comment]
I would like to exclude encrypted files which are named as `**/vault_*.yml` and are not valid yaml files unless decrypted.

**Comment by Hipska** [comment]
I would like to exclude only a specific rule in a specific file using the `.ansible-lint-ignore` file.

Example:
```
roles/*/tasks/main.yml name[missing]
```

Since that file only includes other tasks, no need to name that I guess?
